pkg/utils/utils.go

```diff
@@ -74,6 +80,7 @@ func init() {
 		logrus.Fatalf("OPENEBS_NODE_ID not set")
 	}
 
+	GoogleAnalyticsEnabled = os.Getenv(GoogleAnalyticsKey)
```

Comment:
Have we considered the cases for invalid values for the env variable... 
One way -- 
1. If the value is valid -- all good
2. If the value is invalid -- set the google analytics to `disabled` mode ( or whatever is the default mode is env is not provided ? ) and log this thing in a message.

Reply:
if ENV value is  only `true` then analytics event will be sent, for any other values no event will be pushed.